tests/roles/mariadb_copy/templates/restore_dbs.bash

```diff
@@ -37,9 +37,15 @@ for db_file in ${all_db_files}; do
     fi
     echo "creating ${db_name} in ${db_server}"
     container_name=$(echo "mariadb-client-${db_name}-create" | sed 's/_/-/g')
+    # ironic-inspector DB name uses underscore in ironic-operator
+    if [ ${db_name} == "ironic-inspector" ]; then
+      db_name=${db_name/-/_}
+    fi
```

Comment:
This looks like the db_name_map declared on line 9 could be used for it? We're already renaming other DBs. Also we should add it to the docs too:

https://github.com/openstack-k8s-operators/data-plane-adoption/blob/main/docs_user/modules/openstack-mariadb_copy.adoc?plain=1#L147

Reply:
Thanks for pointing that out. It should go in the db_name_map.

Done.